readthedocs/projects/tasks/utils.py

```diff
@@ -323,6 +323,160 @@ def deprecated_config_file_used_notification():
     )
 
 
+class DeprecatedBuildImageSiteNotification(SiteNotification):
+    failure_message = _(
+        'Your project(s) "{{ project_slugs }}" are using the deprecated "build.image" '
```

Comment:
Would be nice to link to these projects, I've found it annoying not to be able to click the link to the project, but not a huge deal.

Reply:
Our notification system doesn't support links in this way. It has a JS that if you click on a link from a notification it marks it as read 🤷🏼 -- which is unfortunate for the use case you are suggesting. However, I agree with you that's not great 😄

Reply:
Something for the notification refactor 🤔